manifests/server/grant.pp

```diff
@@ -449,9 +452,9 @@
 
   $_onlyif = $onlyif_function ? {
     'table_exists'    => "SELECT true FROM pg_tables WHERE tablename = '${_togrant_object_only}'",
-    'language_exists' => "SELECT true from pg_language WHERE lanname = '${_togrant_object_only}}'",
+    'language_exists' => "SELECT true from pg_language WHERE lanname = '${_togrant_object_only}'",
     'role_exists'     => "SELECT 1 FROM pg_roles WHERE rolname = '${role}' or '${role}' = 'PUBLIC'",
-    'function_exists' => "SELECT true FROM pg_proc WHERE (oid::regprocedure)::text = '${_togrant_object_only}}${arguments}'",
+    'function_exists' => "SELECT true FROM pg_proc WHERE (oid::regprocedure)::text = '${_togrant_object_only}${arguments}'",
```

Comment:
This is not covered by the commit message and seems to be an interpolation fix.  Was it intended to be fixed in another commit / PR?

Reply:
Yes, yes it was... Thanks for bringing it to my attention @smortex.
I've corrected